FFMpegCore/FFMpeg/Arguments/RotateArgument.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace FFMpegCore.Arguments
+{
+    /// <summary>
+    /// Represents size parameter
+    /// </summary>
+    public class RotateArgument : IArgument
```

Comment:
See VideoFiltersArgument and TransposeArgument
This functionality already exists